src/SFA.DAS.CaptureTheFlag.Web/wwwroot/js/Lobby.js

```diff
@@ -0,0 +1,45 @@
+﻿'use strict'
+
+window.setInterval(function () {
+
+    var url = "/lobby/refresh?gameId=" + pageModel.Id
+    
+    var httpRequest = new XMLHttpRequest();
+
+    httpRequest.open("GET", url, false);
+
+    httpRequest.send();
+    
+    UpdateLobby(JSON.parse(httpRequest.responseText))
+
+}, 1500);
```

Comment:
Again, this smells but I'm not sure why we're doing it. Maybe it needs a re-think?